worker/caasadmission/handler.go

```diff
@@ -109,10 +109,24 @@ func admissionHandler(logger Logger, rbacMapper RBACMapper) http.Handler {
 			return
 		}
 
+		logger.Infof("recieved admission request for %s of %s in namespace %s",
+			admissionReview.Request.Name,
+			admissionReview.Request.Kind,
+			admissionReview.Request.Namespace,
+		)
+
 		reviewResponse := &admission.AdmissionResponse{
 			Allowed: true,
 		}
 
+		for _, ignoreObjKind := range admissionObjectIgnores {
+			if compareAPIGroupVersionKind(ignoreObjKind, admissionReview.Request.Kind) {
```

Comment:
We should have a unit test for this filtering out

Reply:
Done & Done